json-schema/json-schema-generic/src/test/scala/endpoints/generic/JsonSchemasTest.scala

```diff
@@ -25,32 +28,65 @@ class JsonSchemasTest extends FreeSpec {
     }
   }
 
-  object DocumentedGenericSchemas extends GenericSchemas with openapi.JsonSchemas
+  object FakeAlgebraJsonSchemas extends GenericSchemas with endpoints.algebra.JsonSchemas {
```

Comment:
Dependency on `openapi.JsonSchemas` was cause of cyclic dependency when I wanted to depend on `json-schema-generic` module in openapi tests. Couldn't break this cycle in SBT, so decided to define some trivial, string-based test interpreter here in order to break dependency on `openapi`, which seemed weird anyway.